cli/config/src/lib.rs

```diff
@@ -64,8 +64,20 @@ impl Config {
             Some(location) => location,
             None => return Config::initial(),
         };
-        let content = fs::read_to_string(&location)?;
-        let config = serde_json::from_str(&content)?;
+        let content = fs::read_to_string(&location).or_else(|err| {
+            Err(anyhow!(
```

Comment:
I think you could use anyhow's [`with_context`](https://docs.rs/anyhow/1.0.42/anyhow/trait.Context.html#tymethod.with_context) method to express this slightly more concisely:

```rust
let content = fs::read_to_string(&location)
    .with_context(|| format!("failed to read {}", location.to_string_lossy()))?;
// ...
```

Reply:
Thank you for the suggestion! I've fixed it.